contracts/RandomBeaconHistory.cdc

```diff
@@ -0,0 +1,90 @@
+/// RandomBeaconHistory (FLIP 123)
+///
+/// This contract stores the history of random sources generated by the Flow network. The defined Heartbeat resource is
+/// updated by the Flow Service Account at the end of every block with that block's source of randomness.
+///
+/// While the source values are random, generated by the Random Beacon and transmitted into the execution 
+/// environment via the committing transaction, using the raw values from this contract does not guarantee secure
+/// randomness. The Hearbeat is intended to be used in conjunction with a pseudo-random number generator (PRNG)
+/// construct and commit-reveal mechanism to provide an onchain source of secure, non-revertible randomness.
```

Comment:
IMO the current formulation suggests that using a PRG makes the random source more secure. The PRG is only useful to extend the small random source into a longer random sequence.
I also added minor suggestions, feel free to update the wording.

```suggestion
/// While the source values are safely generated by the Random Beacon (non-predictable, unbiasable, verifiable) and transmitted into the execution 
/// environment via the committing transaction, using the raw values from this contract does not guarantee non-revertible
/// randomness. The Hearbeat is intended to be used in conjunction with a
/// commit-reveal mechanism to provide an onchain source of non-revertible randomness.
// It is also recommended to use the source values with a pseudo-random number 
// generator (PRNG) to generate an arbitrary-long sequence of random values. 
// 
// For usage of randomness where result abortion is not an issue, it is recommended
// to use the Cadence built-in function `revertibleRandom`, which is also based on 
// the safe Random Beacon. 
```

Reply:
Ah I see, thanks for the clarification!